tests/auth_tests/test_forms.py

```diff
@@ -412,6 +412,25 @@ class Meta(BaseUserCreationForm.Meta):
         user = form.save(commit=True)
         self.assertSequenceEqual(user.orgs.all(), [organization])
 
+    def test_custom_form_with_non_required_password(self):
+
+        class CustomUserCreationForm(BaseUserCreationForm):
+
+            another_field = forms.CharField(required=True)
```

Comment:
```suggestion
        class CustomUserCreationForm(BaseUserCreationForm):
            another_field = forms.CharField(required=True)
```
(nit - looks like other forms have less spacing)

Or another option would be:
```python
        class CustomUserCreationForm(BaseUserCreationForm):
            password1 = forms.CharField(required=False)
            password2 = forms.CharField(required=False)
            another_field = forms.CharField(required=True)
```

Reply:
Great suggestion, added (originally I was following the exact same pattern as reported via ticket).